highfidelity/hifi
# Solve Jitteriness when scrolling the content in HMD mode -> Marketplace  using the scrollbar.

Fix for Bug WL 21298 : Marketplace scrolling in HMD is jittery when using the scrollbar

**# QA To Test**

HMD / Handcontrollers required

1. Put on HMD.
2. Open Tablet UI
3. Select Marketplace
4. Scroll the content. [Select any category]

**Expected Result** : You will not see any kind of jitteriness while scrolling the content. Scrolling is smooth now.

_Note: Dev tested on Windows 10 + Oculus Rift_
